roles/container-engine/docker/tasks/main.yml

```diff
@@ -105,9 +102,14 @@
   when: ansible_distribution == "Fedora" and not is_atomic
 
 - name: Configure docker repository on RedHat/CentOS/Oracle Linux
-  template:
-    src: "rh_docker.repo.j2"
-    dest: "{{ yum_repo_dir }}/docker.repo"
+  yum_repository:
+    name: docker-ce
+    baseurl: "{{ dockerproject_rh_repo_base_url }}"
+    description: "Docker CE Stable - $basearch"
+    gpgcheck: yes
+    gpgkey: "{{ dockerproject_rh_repo_gpgkey }}"
+    keepcache: "{{ docker_rpm_keepcache | default('1') }}"
+    proxy: " {{ http_proxy | default('_none_') }}"
```

Comment:
This is wrong, `rh_docker.repo.j2` contains `module_hotfixes=True` for RHEL/CentOS 8 that you just dropped.